playbook/appherd/roles/django_export_model/tasks/main.yml

```diff
@@ -6,7 +6,32 @@
 # Perform django management command to export a model as csv
 # add in group create for first run only.
 
-- name: "Django export {{ export_model }}"
+- name: "clean up old export csv copies"
+  become_user: "{{ remote_admin_account }}"
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ cf_app_log_dir }}/bb_{{ env }}_{{ export_app_model }}_list.csv"
+    - "{{ cf_app_log_dir }}/bb_{{ env }}_{{ export_dev_model }}_list.csv"
+
+
+- name: "Django export {{ export_app_model }} CSV"
+  become_user: "{{ remote_admin_account }}"
+  become: yes
+  # We only want to run the command on one server in the fleet
+  run_once: yes
+  shell: |
+    cd {{ cf_app_pyapp_home }}/{{ common_project_name }}
+    source {{ cf_app_py_virtual_env }}/bin/activate
+    export PATH={{ cf_app_py_virtual_env }}/bin:$PATH
+    ./manage.py model2csv --application {{ export_application }} --model {{ export_app_model }} --add_table_name True --filter_fields id,client_id,user,name,agree,created,updated,client_uri,active | sed '/WARNING:/d' | sed 's/{{ export_application }}.{{ export_app_model }}/{{ export_application }}.{{ export_app_model }}_info/' >{{ cf_app_log_dir }}/bb_{{ env }}_{{ export_app_model }}_list.csv
```

Comment:
Are the `user` and `name` fields considered sensitive information in this context?